sphinx/themes/haiku/layout.html

```diff
@@ -51,7 +51,7 @@ <h2 class="heading"><span>{{ title|striptags|e }}</span></h2>
       <div class="topnav" role="navigation" aria-label="Top">
       {{ nav() }}
       </div>
-      <div class="content" role="main">
+      <div class="content" role="main" id="main">
```

Comment:
For this theme and a few others we need to duplicate the node ID because they override the default HTML from `basic` a bit and the ID gets lost in the override.